codegen/config/binder.go

```diff
@@ -520,6 +520,7 @@ func (b *Binder) CopyModifiersFromAst(t *ast.Type, base types.Type) types.Type {
 }
 
 func IsNilable(t types.Type) bool {
+	t = code.Unalias(t)
```

Comment:
Maybe we should call `types.Unalias(t)` so we'll Unalias even in Go 1.22?
Currently in Go 1.22 calling this with a nilable alias type returns false.
@StevenACoffman, @a8m WDYT?

Reply:
We really need to continue to support Go 1.22, and I don't think we can just assume people have gotypesalias=1

Reply:
So I'll call `types.Unalias()` here, that way we'll make sure `IsNilable()` is reliable no matter what.
Also - we don't need the pointer base type unaliasing from `code.Unalias()` here.